bitnami/redis/Chart.yaml

```
@@ -25,4 +25,4 @@ name: redis
 sources:
   - https://github.com/bitnami/bitnami-docker-redis
   - http://redis.io/
-version: 15.5.5
+version: 15.5.6
```

Please bump the minor version instead since you're adding a new parameter:
```suggestion
version: 15.6.0
```